forem/forem
# /comments API docs

## What type of PR is this? (check all applicable)


## Description

Adds the `/comments` API docs to rswag specs

## Related Tickets & Documents



- closes #18742

## QA Instructions, Screenshots, Recordings

Open `swagger/v1/api_v1.json` on https://editor.swagger.com to check it out. Run this locally to QA if you prefer too:

```
SWAGGER_DRY_RUN=0 RAILS_ENV=test rails rswag PATTERN="spec/requests/api/v1/**/*_spec.rb"
```

### UI accessibility concerns?

N/A

## Added/updated tests?


## [Forem core team only] How will this change be communicated?

- [x] This change does not need to be communicated, and this is why not: Ongoing effort to document API V1 will be communicated as a whole when ready

## [optional] What gif best describes this PR or how it makes you feel?

![nuovo articulo](https://media.giphy.com/media/znjASTOGZEDexVwcGL/giphy.gif)
